<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdministratorUI</class>
 <widget class="QDialog" name="AdministratorUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>华工教务 管理员平台</string>
  </property>
  <widget class="QLabel" name="awelcomelabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="Line" name="aline">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>1081</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="aexitButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>文件名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="outButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>200</y>
     <width>20</width>
     <height>661</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>课程信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>200</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>用户信息</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="coursetableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>591</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="usertableWidget">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>240</y>
     <width>421</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="f5Button">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
